lib/storage/metadata/proxy/BucketdRoutes.js

```diff
@@ -106,6 +106,18 @@ class BucketdRoutes {
                 sendResponse(req, res, logger, err, data));
     }
 
+    // `admin' context methods
+
+    _checkHealth(req, res, logger) {
+        return this._metadataWrapper.checkHealth(logger, (err, resp) => {
+            if (err) {
+                logger.error('Failed the health check', { error: err });
```

Comment:
you might want to specify the method in the error log

```suggestion
                logger.error('Failed the health check', { error: err, method: "_checkHealth" });
```